src/updatesemitter.cpp

```diff
@@ -39,6 +40,10 @@ UpdatesEmitter::UpdatesEmitter()
                                                       this)) {
         qCWarning(lcCommHistory) << Q_FUNC_INFO << ": error registering object";
     }
+    if (!QDBusConnection::sessionBus().registerService(QString::fromLatin1("%1.p%2").arg(COMM_HISTORY_SERVICE_NAME).arg(QCoreApplication::applicationPid()))) {
```

Comment:
Could be nice limiting the line length a bit here. Up for consideration is pid best with potentially different namespaces, but low chance of being a real problem.

Reply:
Actually, I noticed that I forgot to unregister the service on the bus. So I'm now saving the service name into a global variable, which has the consequence to reduce the line length !

qtconatcts-sqlite is using a UUID to identify the service. It's a good idea, but it also requires to post-process a bit the UUID string to comform to the D-Bus name specifications. And it gives very long service names, which is not nice when observing the bus and debugging.